### PR TITLE
node: add French translation

### PR DESCRIPTION
**Pull request by Nico385412**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

 - **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/node.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# node
+
+> Plateforme Javascript côté serveur.
+> Plus d'informations : <https://nodejs.org>.
+
+- Éxecute un fichier JavaScript:
+
+`node {{chemin/vers/le/fichier}}`
```
Comment (navarroaxel) [inline pages.fr/common/node.md:8]: I think that we use this form:
```suggestion
`node {{chemin/vers/fichier}}`
```

File: pages.fr/common/node.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`node -e "{{code}}"`
+
+- Évalue et affiche le résultat, très utile pour voir les versions de dépendances node:
+
+`node -p "{{process.versions}}"`
+
+- Active l'inspecteur, mets en pause l'éxécution jusqu'à ce qu'un debugger soit connecté une fois que le code source est totalement interprété:
+
+`node --no-lazy --inspect-brk {{path/to/file}}`
```
Comment (navarroaxel) [inline pages.fr/common/node.md:24]:
```suggestion
`node --no-lazy --inspect-brk {{chemin/vers/fichier}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

File: pages.fr/common/node.md (hunk 2 above)
Comment (Nico385412) [reply pages.fr/common/node.md]: done 👍

File: pages.fr/common/node.md (hunk 1 above)
Comment (Nico385412) [reply pages.fr/common/node.md]: done 👍

**Review by navarroaxel — APPROVED** [review]

**Review by marchersimon — APPROVED** [review]
👍🏽

**Comment by marchersimon** [comment]
@Nico385412 would you like to help out with #6509? We'd just need a translation of a single sentence. Thanks

**Comment by Nico385412** [comment]
> @Nico385412 would you like to help out with #6509? We'd just need a translation of a single sentence. Thanks

done ;) 
https://github.com/tldr-pages/tldr/pull/6823

**Pull request merged**